docs/_docs/continuous-integration/circleci.md

```diff
@@ -83,6 +83,75 @@ deployment:
       - rsync -va --delete ./_site username@my-website:/var/html
 ```
 
+for CircleCI v2, a Docker-based system which new projects will follow, set the `S3_BUCKET_NAME` environment variable (an example of the required config file is shown below).
+
+```yaml
+defaults: &defaults
+  working_directory: ~/repo
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    docker:
+      - image: circleci/ruby:2.5
+    environment:
+      BUNDLE_PATH: ~/repo/vendor/bundle
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - rubygems-v1-{{ checksum "Gemfile.lock" }}
```

Comment:
@tomharvey Can you shed some light on the `rubygems-v1-{{ checksum "Gemfile.lock" }}` variable?
I get the following warning when I try to build the docs on `master`
```
Liquid Warning: Liquid syntax error (line 99): Expected end_of_string but found string in
"{{ checksum "Gemfile.lock" }}"
```
/cc @DirtyF

Reply:
The use of `{{ checksum "Gemfile.lock" }}` in the circleci.yml file will tell circleci to get the checksum of that file.

See https://circleci.com/docs/2.0/caching/

That checksum is then used to as a cache key for a gemset. This way, circleci will only install gems the first time, or it will add new gems when Gemfile.lock changes.

This speeds the use of circleCI considerably.

Reply:
Thanks for getting back, @tomharvey 
For future reference, always enclose Liquid-like constructs within `{% raw %} {% endraw %}`

@DirtyF  please push corrections to `master` : (`docs/_docs/continuous-integration/circleci.md` `ln#103` and `ln#117`)

```diff
- - rubygems-v1-{{ checksum "Gemfile.lock" }}
+ - rubygems-v1-{% raw %}{{ checksum "Gemfile.lock" }}{%endraw%}
```
```diff
- key: rubygems-v1-{{ checksum "Gemfile.lock" }}
+ key: rubygems-v1-{% raw %}{{ checksum "Gemfile.lock" }}{% endraw %}
```

Reply:
What command are you using to build docs - just noticed that those tags are just ignored in the production release.

Happy to push a PR with the raw tags. Sorry.